### PR TITLE
many: progress bars should use the overridable stdouts

### PR DESCRIPTION
**Pull request by pedronis**
making progress bars should take a stdout io.Writer

ToolingStore should carry an overridable stdout

make sure in image, preseed and cmd/snap code the overridable
top-level Stdout is passed along to make progress bars

### CODE REVIEW

**Comment by jawn-smith** [comment]
@pedronis I can confirm this gives `ubuntu-image` the flexibility to suppress or allow the progress bars depending on verbosity leve. Thanks!

**Review by mvo5 — APPROVED** [review]
Thank you!

File: store/tooling/tooling.go [hunk 1]
```diff
@@ -49,6 +50,10 @@ import (
 
 // ToolingStore wraps access to the store for tools.
 type ToolingStore struct {
+	// Stdout is for output, mainly progess bars
```
Comment (MiguelPires) [inline store/tooling/tooling.go:53]:
```suggestion
	// Stdout is for output, mainly progress bars
```

**Review by MiguelPires — APPROVED** [review]
LGTM

**Pull request merged**